src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedBackingFieldSymbol.cs

```diff
@@ -18,11 +19,22 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal sealed class SynthesizedBackingFieldSymbol : FieldSymbolWithAttributesAndModifiers
     {
+        [Flags]
+        private enum Flags
+        {
+            HasInitializer = 1 << 0,
+            IsCreatedForFieldKeyword = 1 << 1,
+            IsEarlyConstructed = 1 << 2,
+        }
+
         private readonly SourcePropertySymbolBase _property;
         private readonly string _name;
-        internal bool HasInitializer { get; }
-        internal bool IsCreatedForFieldKeyword { get; }
-        internal bool IsEarlyConstructed { get; }
+        private readonly Flags _backingFieldFlags;
```

Comment:
>private readonly Flags _backingFieldFlags; [](http://example.com/codeflow?start=8&length=42)

It looks like this refactoring is not related to the goal of this PR. Please extract into a separate PR.